src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -206,7 +223,9 @@ export class SearchMenu extends React.Component {
                       }
                       {...getItemProps({ item: suggestion })}
                       // eslint-disable-next-line react/no-danger
-                      dangerouslySetInnerHTML={{ __html: formattedSuggestion }}
+                      dangerouslySetInnerHTML={{
+                        __html: formattedSuggestion,
+                      }}
```

Comment:
A little out of scope of this PR but I just noticed this `dangerouslySetInnerHTML` to inject the `<strong>` tag into the suggestion. I think we can get past that by doing something like -

```jsx
{this.state.suggestions?.map((suggestion, index) => {
const tail = suggestion.slice(this.state.userInput.length);
const formattedSuggestion = (
  <>
    <strong>{this.state.userInput}</strong>{tail}
  </>
)
// ...
```

It doesn't seem like the search.gov API returns any results for any HTML input but might still be better to be defensive just in case

Reply:
I was concerned with your previous statement about what happens if the userInput actually appears in the middle of the suggestion

```
const splitSuggestion = suggestion.split(this.state.userInput)
<>
splitSuggestion[0] <strong>this.state.userInput</strong>splitSuggestion[1]
</>
```

Reply:
Also, I had a conversation with Kelson about this, and he suggested the current use of lodash escape to help mitigate any issues that might arise here.

Reply:
Definitely lots of ways to fix this @kelsonic and @ncksllvn, going to be conversations to be had for updates prior to type ahead 1.0